assets/js/admin.js

```diff
@@ -0,0 +1,43 @@
+$ = jQuery;
+
+$(document).ready(function() {
+
+	/**
+	 * Taxonomy_Image
+	 */
+	$("#insert_image_tag_button").off("click").on("click", function () {
+		if ( typeof wp !== "undefined" ) {
+			var media_modal = wp.media({
+				title: "Insert Image",
+				library: {
+					type: "image"
+				},
+				button: {
+					text: "Insert"
```

Comment:
@mcshanea @tcrsavage Are these need to be registered for translations, or we're not bothering with admin side for translations?